axon-server/administration/error-codes.md

```diff
@@ -0,0 +1,658 @@
+# Error Codes
+
+This page depicts the various Error codes that Axon Server will return in the case of any problems while Processing Client Requests / Message Handling / Administrative Tasks / Cluster Errors.
+
+<table>
+  <thead>
+    <tr>
+      <th style="text-align:left">Area</th>
+      <th style="text-align:left">Error Code</th>
+      <th style="text-align:left">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td style="text-align:left"><em>Client Request Processing Errors</em>
+      </td>
+      <td style="text-align:left"></td>
+      <td style="text-align:left"></td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1000</td>
+      <td style="text-align:left">
+        <p><b>AUTHENTICATION_TOKEN_MISSING</b>
+        </p>
+        <p>This indicates that the Axon Server has been configured with authentication
+          enabled and will reject any instructions from a client application if the
+          token is missing.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1001</td>
+      <td style="text-align:left">
+        <p><b>AUTHENTICATION_INVALID_TOKEN</b>
+        </p>
+        <p>This indicates that the Axon Server has been configured with authentication
+          enabled and will reject any instructions from a client application if an
+          invalid token has been supplied.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1002</td>
+      <td style="text-align:left">
+        <p><b>UNSUPPORTED_INSTRUCTION</b>
+        </p>
+        <p>This error is returned when the Axon Server cannot recognize the instruction
+          passed to it.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1003</td>
+      <td style="text-align:left">
+        <p><b>INSTRUCTION_EXECUTION_ERROR</b>
+        </p>
+        <p>This error is returned when the Axon Server throws an error during the
+          execution of the instruction passed to it.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1004</td>
+      <td style="text-align:left">
+        <p><b>INSTRUCTION_RESULT_TIMEOUT</b>
+        </p>
+        <p>This indicates that the Axon Server threw an error during the execution
+          of the instruction passed to it.</p>
```

Comment:
"This indicates that the execution of the instruction did not finish within the timeout set." I do not know if we have a property for this timeout.